app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdUpdate.kt

```diff
@@ -100,7 +102,11 @@ class ScanSimpleIdUpdate @Inject constructor(
       event: PatientSearchByIdentifierCompleted
   ): Next<ScanSimpleIdModel, ScanSimpleIdEffect> {
     return if (event.patients.isEmpty()) {
-      dispatch(OnlinePatientLookupWithIdentifier(event.identifier))
+      if (isOnlinePatientLookupEnabled) {
+        dispatch(OnlinePatientLookupWithIdentifier(event.identifier))
+      } else {
+        dispatch(OpenPatientSearch(event.identifier, null, model.patientPrefillInfo))
+      }
```

Comment:
Since we are just dispatching effects here. We can extract out the effect variable.

```suggestion
val effect = if (isOnlinePatientLookupEnabled)
  OnlinePatientLookupWithIdentifier(event.identifier)
else
  OpenPatientSearch(event.identifier, null, model.patientPrefillInfo)
```